### ISSUE TITLE
OpenPNM and porespy are cyclically dependent

### ISSUE THREAD

**Issue by ma-sadeghi** [PMEAL/OpenPNM#1228]
`OpenPNM` is listed as a dependency for `porespy` and vice versa. It's not a major problem, but I'm not sure it's a good practice either.

**Comment by TomTranter** [comment]
Yep, this was done as some of the network extraction functions were written in OpenPNM already. I don't think it really matters but could conceivably refactor stuff to avoid it. Also Porespy has examples that use OpenPNM and probably visa versa so it may not be worth worrying about
